packages/@aws-cdk/aws-glue/README.md

```diff
@@ -1,2 +1,104 @@
 ## The CDK Construct Library for AWS Glue
 This module is part of the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk) project.
+
+### Database
+
+A `Database` is a logical grouping of `Tables` in the Glue Catalog.
+
+```ts
+new glue.Database(stack, 'MyDatabase', {
+  databaseName: 'my_database'
+});
+```
+
+By default, a S3 bucket is created where the Database is stored under  `s3://<bucket-name>/`, but you can manually specify another location:
```

Comment:
`<bucket-name>` or `<database-name>`?

Reply:
the name of the bucket is generated and not related to the database name.